app/javascript/packs/webauthn-unhide-signin.js

```diff
@@ -4,14 +4,14 @@ function unhideWebauthn() {
   if (WebAuthn.isWebAuthnEnabled()) {
     const elem = document.getElementById('select_webauthn');
     if (elem) {
-      elem.classList.remove('hide');
+      elem.classList.remove('display-none');
```

Comment:
This is for alternative options for "Choose another authentication method", to present Security Key as an option only if the browser supports it.

Suggesting to keep this as `display: none`, since it's intended not to be available as an option by default.